code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -718,7 +720,7 @@
 		qdel(src)
 
 /obj/item/weapon/reagent_containers/food/drinks/soda_cans/proc/pop_open(var/mob/user)
-	to_chat(user, "You pull back the tab of \the [src] with a satisfying pop.")
+	to_chat(user, "[tabself] of \the [src] with a satisfying pop.")
```

Comment:
This "of" is unneeded, since the tabself variable has the ofs already.

Reply:
was pinged about it on the discord but thx